Sources/Diagnostics/DiagnosticsEvent.swift

```diff
@@ -81,6 +81,7 @@ struct DiagnosticsEvent: Codable, Equatable {
     struct Properties: Codable, Equatable {
         let verificationResult: String?
         let endpointName: String?
+        let host: String?
```

Comment:
We need to add this to the backend schema as well (should just be ignored until then). Lmk if you need any help with this!!

Reply:
Yep! Before doing the work, I wanted to make sure that this is the implementation (and parameter name) we wanted. Thanks for flagging!